### DISCUSSION TITLE
Can I clone this project for my envirolment?

### DISCUSSION THREAD

**Discussion by marcelorcastro** [MichaelCade/90DaysOfDevOps#104] [General]
Hi, first all, congratulations for this job!

This was exactly my 'day by day'.

My question is: can I clone this repository in my Gitlab and edit then for use GCP? I want to use this for create a devops team.

Thanks!

**Comment by MichaelCade** [comment]
Hey, 
Thanks, 

By all means, according to the contributions and code of conduct you are able to take a copy and use it in any of the allowed ways. 

Would love for this to be used to educate your team. Just note that this repository is not complete and we will be adding more to it. It might make sense to use this as your source.
